### DISCUSSION TITLE
New maintainer

### DISCUSSION THREAD

**Discussion by sckott** [ropensci/pangaear#77] [General]
@gavinsimpson @karawoo @aammd @naupaka   Any interest taking over maintainer role? I can't do it any more

**Comment by naupaka** [comment]
@sckott maybe — is there a place to read more about what’s involved to maintain an ROpenSci package?

**Reply by sckott** [reply]
Here's the devguide https://devguide.ropensci.org/ - not too different from any normal pkg, just recommended best practices mostly.

**Reply by naupaka** [reply]
Yes @sckott I can take this on. I am happy to do my best to get up to speed on the ROpenSci repo maintainer documentation. I am traveling until mid July but can dive in after that if there are particular things needing done.

**Reply by sckott** [reply]
Great, thanks! You and Gavin and other maintainers have admin access now

**Comment by gavinsimpson** [comment]
I can certainly help out with this too, just don't want to commit to another new thing while I'm starting a new job having taken on a couple of new responsibilities recently.

**Reply by sckott** [reply]
Thanks Gavin!
